src/components/common/unlock-button/unlock-button.scss

```diff
@@ -1,10 +1,10 @@
 .unlock-button {
   @apply mvx:pl-3 mvx:pr-4 mvx:h-15 mvx:flex! mvx:gap-4 mvx:cursor-pointer mvx:transition-all mvx:duration-200 mvx:items-center;
   @apply mvx:ease-in-out;
-  background: rgba(53, 53, 53, 0.3);
```

Comment:
We need to change the approach here, it's not correct to hold opacity settings in color variables.